Sources/tuist/tuist.docc/Tutorials/Tuist Tutorial/Getting-Started/Add External Dependencies/Tuist Tutorial - Add External Dependencies.tutorial

```diff
@@ -0,0 +1,92 @@
+@Tutorial(time: 5) {
+    @Intro(title: "Adding external Dependencies") {
+        Learn how to integrate external dependencies in a Tuist.
+
+        Integrating external dependencies is an important but often painful part of Xcode project management. Dependency managers like [CocoaPods](https://cocoapods.org) integrate it when running `pod install` leveraging Xcode workspaces, and [Swift Package Manager](https://www.swift.org/package-manager) does it at build time leveraging Xcode closed build system. Both approaches might lead to integration issues that can cause compilation issues down the road.
```

Comment:
What about shortening this one? I'd use the intro to describe what the tutorial is about, and if people are interesting in knowing more about how the different package managers compare with each other, we can write a documentation page for that (as part of another PR).

<img width="719" alt="image" src="https://github.com/tuist/tuist/assets/663605/81bb130a-b8aa-425c-9ba6-a39e5f913b7f">